src/RequestFactory.php

```diff
@@ -54,6 +54,35 @@ public function create($serviceKey, $requestKey, array $parameters = [])
     }
 
     /**
+     * Builds a request based on the service and request name, using provided $jsonSerializable as request body
+     *
+     * @param string            $serviceKey
+     * @param string            $requestKey
+     * @param \JsonSerializable $jsonSerializableBody
+     * @return mixed
+     */
+    public function createFromJson($serviceKey, $requestKey, \JsonSerializable $jsonSerializableBody) {
```

Comment:
Why do we need an specific function for JsonSerializable that checks that the 'application/json' header is set if the other function `create` parses to JSON if this header is set? Also, with this method you are not replacing the placeholders of the body, while the other function does.